src/decoder.rs

```diff
@@ -505,10 +505,21 @@ impl<R: BufRead + Seek> WebPDecoder<R> {
         self.memory_limit = limit;
     }
 
+    /// Get the background color specified in the image file if the image is extended and animated webp.
+    pub fn get_background_color_hint(&self) -> Result<[u8; 4], DecodingError> {
```

Comment:
I think this should be called `background_color_hint` (our other metadata methods don't start with `get_`) and should return an `Option<[u8; 4]>` rather than a result.